src/main/java/core/basesyntax/service/impl/FruitTransactionMapper.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Operation;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitTransactionMapper implements ParserService<String> {
+    private static final int OPERATION_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final String DIVIDER = ",";
```

Comment:
Please decide how are you going to call this literal DIVIDER  or SEPARATOR (like in ReportBuilderImp), I suggest DELIMITER